frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerLifeCycle.java

```diff
@@ -39,6 +39,43 @@ public Process getProcess() {
         return process;
     }
 
+    public static ArrayList<String> launcherArgsToList(String launcherArgs) {
+        ArrayList<String> arrlist = new ArrayList<String>();
+        arrlist.add("-m");
+        arrlist.add("intel_extension_for_pytorch.cpu.launch");
+        arrlist.add("--ninstance");
+        arrlist.add("1");
+        if (launcherArgs != null && launcherArgs.length() > 1) {
+            String[] argarray = launcherArgs.split(" ");
+            for (int i = 0; i < argarray.length; i++) {
+                arrlist.add(argarray[i]);
+            }
+        }
+        return arrlist;
+    }
+
+    public static boolean isLauncherAvailable(String launcherArgs) {
+        boolean launcherAvailable = false;
+        try {
+            ArrayList<String> cmd = new ArrayList<String>();
+            cmd.add("python");
+            ArrayList<String> args = launcherArgsToList(launcherArgs);
+            cmd.addAll(args);
+            cmd.add("--no_python");
```

Comment:
How does this work exactly? We have two commands `python` and `--no_python`?

Reply:
Launcher uses the following command line:
`python -m intel_extension_for_pytorch.cpu.launch [knobs] <your_pytorch_script> [args]`
The `python` in line 61 is for the launch script and the `--no_python` is for running non Python script with the launcher. Here, we want to test if launcher is available with just a dummy command line, `hostname`, hence `--no_python`.

Reply:
Sure, will add a comment. Only `IPEX` and `numactl` needs to be installed, which are both checked in `isLauncherAvailable`.